src/components/MoneyReportHeaderStatusBar.tsx

```diff
@@ -16,19 +20,31 @@ type MoneyReportHeaderStatusBarProps = {
 function MoneyReportHeaderStatusBar({nextStep}: MoneyReportHeaderStatusBarProps) {
     const styles = useThemeStyles();
     const {translate} = useLocalize();
+    const theme = useTheme();
 
     const messageContent = useMemo(() => {
         const messageArray = nextStep.message;
         return NextStepUtils.parseMessage(messageArray);
     }, [nextStep.message]);
 
+    const HOURGLASS_ICON = 'hourglass';
+
     return (
         <View style={[styles.dFlex, styles.flexRow, styles.alignItemsCenter, styles.overflowHidden, styles.w100, styles.headerStatusBarContainer]}>
             <View style={[styles.mr3]}>
-                <Badge
-                    text={translate(nextStep.title === CONST.NEXT_STEP.FINISHED ? 'iou.finished' : 'iou.nextStep')}
-                    badgeStyles={styles.ml0}
-                />
+                {nextStep?.icon && nextStep.icon === HOURGLASS_ICON ? (
```

Comment:
We should at least account for the other potential icons here - "checkmark" and "stopwatch", since those will be coming through soon